### DISCUSSION TITLE
Random Pause on S1 plus

### DISCUSSION THREAD

**Discussion by mattfox27** [mriscoc/Ender3V2S1#1037] [General]
So I installed Prof F/W ona s1 plus with non touch screen.  I set the min/max and bed size to 300x300 however when I print something it will lay down the skirts then start the first layer, after like 20 lines it just randomly stops, parks and unloads.

Using cura sand g code as before with touchscreen that worked.

**Comment by mriscoc** [comment]
Disable your filament runout sensor: https://github.com/mriscoc/Ender3V2S1/issues/1025
